server/routes/createCommunity.ts

```diff
@@ -33,6 +42,20 @@ const createCommunity = async (models, req: Request, res: Response, next: NextFu
   if (req.body.invitesEnabled !== 'true' && req.body.invitesEnabled !== 'false') {
     return next(new Error(Errors.NoInvitesEnableldSetting));
   }
+
+  const { website, discord, element, telegram, github } = req.body;
+  if (website && !urlHasValidHTTPPrefix(website)) {
+    return next(new Error(Errors.InvalidWebsite));
+  } else if (discord && !urlHasValidHTTPPrefix(discord)) {
```

Comment:
This isn't really sufficient checking for whether a link actually goes to discord/github/element etc, but it's fine for now.

Reply:
yes. exactly. just followed the other parts of the code